server/routes/otherRoutes.ts

```diff
@@ -44,15 +44,19 @@ export default class OtherRoutes {
   public getPrisonerImage: RequestHandler = async (req, res): Promise<void> => {
     const { username } = res.locals.user
     const { nomsId } = req.params
-    this.prisonerService
-      .getPrisonerImage(username, nomsId)
-      .then(data => {
-        res.type('image/jpeg')
-        data.pipe(res)
-      })
-      .catch(() => {
-        const placeHolder = path.join(process.cwd(), '/assets/images/image-missing.png')
-        res.sendFile(placeHolder)
-      })
+    const placeHolder = path.join(process.cwd(), 'assets/images/image-missing.png')
+    if (nomsId !== 'placeholder') {
+      this.prisonerService
+        .getPrisonerImage(username, nomsId)
+        .then(data => {
+          res.type('image/jpeg')
+          data.pipe(res)
+        })
+        .catch(error => {
+          res.sendFile(placeHolder)
+        })
+    } else {
+      res.sendFile(placeHolder)
+    }
```

Comment:
Caters for missing NomsId too - probation cases may never have been into prison.

Reply:
I must remember to copy this!